<ui version="4.0" >
 <class>HelpDialogBase</class>
 <widget class="QDialog" name="HelpDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Help</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="1" >
    <widget class="QPushButton" name="mOkBtn" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QTextEdit" name="mHelpBrowser" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mOkBtn</sender>
   <signal>clicked()</signal>
   <receiver>HelpDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>387</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>182</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
